apps/vmq_http_pub/src/vmq_http_pub_app.erl

```diff
@@ -0,0 +1,27 @@
+%%%-------------------------------------------------------------------
+%% @doc vmq_http_pub plugin public API
+%% @end
+%%%-------------------------------------------------------------------
+
+-module(vmq_http_pub_app).
+
+-behaviour(application).
+
+%% Application callbacks
+-export([start/2, stop/1]).
+
+%%====================================================================
+%% API
+%%====================================================================
+
+start(_StartType, _StartArgs) ->
+    {ok, _} = application:ensure_all_started(vmq_passwd),
```

Comment:
We should not assume any specific authentication plugin.

Reply:
I took the boilerplate from the passwd and acl plugin. This seems to be a leftover...